.github/workflows/build-and-test-toolchain.yml

```diff
@@ -155,6 +160,10 @@ jobs:
         run: |
           cat ${{ steps.get-wsl-paths.outputs.artifact-path }}\gcc-tests-${{ env.TAG }}\summary.txt >> $env:GITHUB_STEP_SUMMARY
 
+      - name: Group failing tests by filepath of their source files
```

Comment:
As we will be improving how the results are gropped the semantics of the script will probably change which will made this step name obsolte. It would be better to name it little bit more generically (and shortly), e.g. `Group test failures`

Reply:
Renamed the step to `Group test failures` and accordingly also the script itself.